src/foam/u2/view/FilteredReferenceView.js

```diff
@@ -0,0 +1,93 @@
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'FilteredReferenceView',
+  extends: 'foam.u2.View',
+
```

Comment:
Please add a `documentation` comment to this model so it's clear to reader's what it does and what its intended use cases are.